Sitka Styling Improvements and Native JS Conversion

taija opened the pull request:
- Roll in previous pull request (never released) with Sitka Spruce styling improvements
- Fix detection of core site homepage for minor severity alerts
- Remove jQuery based JavaScript alert display script in favor of a native JavaScript and Web Components based script
  - Note: in order for this to work it needs to be able to hook into the WordPress body element; this requires fixes to Bellevue 2022 and Sitka Spruce.

taija committed:
🐛 FIX: Site Homepage Detection in Bellevue 2022

Homepage was not being detected properly due to use of custom post type.
All 'Home' post types will now be detected as the homepage.

taija committed:
🐛 FIX: Sanitization of College Open Message

taija committed:
👌 IMPROVE: Re-Write JavaScript

- Add wp-scripts based build pipeline
- Add native JavaScript Web Component for alert
- Re-write alert handling in native JavaScript - no more jQuery!

taija committed:
Sitka Spruce Updates (#6)

* Improve Display of Alerts

Switch from using Bootstrap Alerts to using a colorful background.

* Improve Bellevue 2022 Homepage Detection

* Remove Legacy Open Message Output

taija committed:
🐛 FIX: Prevent Auto-Scroll on Every Fetch

While alert was active, scroll-to-alert was triggering on every alert fetch. Now this only triggers  on alert change.

TaylorYeeBC reviewed (commented):
Code Reviewed. Need to coordinate with Taija for functionality testing.

TaylorYeeBC commented:
When a Manual Alert is active, an Emergency Alert fails to overwrite content of a loaded page.

Example of incorrect content.
<img width="1919" height="581" alt="image" src="https://github.com/user-attachments/assets/e55485df-8d8e-44d3-9aae-1971ba8477c7" />

taija committed:
Merge branch 'release-v1.11' into native-js

taija committed:
🐛 FIX: Alert Display when Manual Alert Active

Emergency Alert content would not display until page refresh if there was a manual alert present.
This has been resolved by restructuring the alert to handle each instead of overriding the container content.

taija committed:
🐛 FIX: Populate Classes Even if Alert is Unchanged

TaylorYeeBC reviewed (approved):
Reviewed again and tested across Douglas Fir, Bellevue 2022, and Sitka Spruce themes on Chrome and Firefox. 👍

efuenzalida52 reviewed (approved):
code review looks good!
